tests/examples/logging/playground/printstuff/run

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+exec 2>&1
```

Comment:
If we require each service to do this right, it will mostly be done wrong.
Need to see if we can make this behavior part of the framework.